src/lib/storage/StorageApi.ts

```diff
@@ -157,20 +164,21 @@ export class StorageApi {
   }
 
   /**
-   * Rename a file
-   * @param bucketName
-   * @param fromPath original relative file path
-   * @param toPath the new relative file path
+   * Moves an existing file, optionally renaming it at the same time.
+   *
+   * @param bucketId The bucket which contains the file.
+   * @param fromPath The original file path, including the current file name. For example `folder/image.png`.
+   * @param toPath The new file path, including the new file name. For example `folder/image-copy.png`.
    */
-  async renameFile(
-    bucketName: string,
+  async moveFile(
+    bucketId: string,
     fromPath: string,
     toPath: string
   ): Promise<{ data: { message: string } | null; error: Error | null }> {
     try {
       const data = await post(
         `${this.url}/object/rename`,
```

Comment:
@inian keep it as `rename` here though right?

Reply:
yup. I will update this after the backend change is done